lk2nd/target_display_cont_splash_mdp3.c

```diff
@@ -0,0 +1,45 @@
+#include <arch/defines.h>
+#include <bits.h>
+#include <debug.h>
+#include <reg.h>
+#include <string.h>
+#include <target.h>
+
+#include <dev/fbcon.h>
+#include <kernel/event.h>
+#include <kernel/thread.h>
+#include <mdp3.h>
+#include <platform.h>
+#include <platform/clock.h>
+#include <platform/iomap.h>
+#include <platform/timer.h>
+
+
+static struct fbcon_config fb = {
+	.bpp    = 24,
+	.format = FB_FORMAT_RGB888,
+};
+
+static void mdss_mdp_cmd_kickoff(void) {
+	writel(0x00000001, MDP_DMA_P_START);
+	mdelay(10);
+}
+
+int mdp3_read_config(struct fbcon_config *fb)
```

Comment:
this function returns a value, you should do some tests (e.g. if `fb->base != NULL)`) like in mdp5.